File: qc-domain/src/main/java/com/eager/questioncloud/domain/creator/CreatorInformation.java
package com.eager.questioncloud.domain.creator;

import com.eager.questioncloud.domain.question.Subject;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class CreatorInformation {
    private Long creatorId;
    private String name;
    private String profileImage;
    private Subject mainSubject;
    private String email;
    private int salesCount;
    private Double rate;
    private String introduction;
}

File: qc-application/src/main/java/com/eager/questioncloud/application/api/creator/dto/CreatorInformation.kt
package com.eager.questioncloud.application.api.creator.dto

import com.eager.questioncloud.core.domain.creator.dto.CreatorProfile

class CreatorInformation(
    val creatorProfile: CreatorProfile,
    val salesCount: Int,
    val averageRateOfReview: Double,
    val subscriberCount: Int,
)
